package.json

```diff
@@ -20,7 +20,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "cypress:open": "cypress open"
```

Comment:
Please modify the script to run the project and cypress form 1 npm run command

Reply:
And good to have  "cypress:run": "cypress run" as well (it is faster to quick run)

Reply:
Done